docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -112,61 +114,107 @@ After the active instance has been configured, you can configure one or more sta
 
 Your standby instance is now configured.
 
+### Verify HA
+
+To confirm communication between the active and standby, you can do the following:
+
+- Click **Make Active** on the standby (but don't promote it). You should see a list of available backups that you can restore from.
+- Verify that Prometheus on the standby is able to see similar metrics to the active. Navigate to `http://<STANDBY_IP>:9090/targets`; the federate target should have a status of UP, and the endpoint should match the active instance IP address.
+
+    ![Verify Prometheus](/images/yp/high-availability/ha-prometheus.png)
+
+- Navigate to **Profile > Releases** and verify that the standby has all the database releases that are in use by universes, and that the releases are listed as Active. To discover all the database releases that are in use by universes, you can view the **Dashboard** page.
+
+    If releases are missing, follow the instructions in [How to configure YugabyteDB Anywhere to provide Older, Hotfix, or Debug Builds](https://support.yugabyte.com/hc/en-us/articles/360054421952-How-to-configure-YugabyteDB-Anywhere-to-provide-Older-Hotfix-or-Debug-Builds).
+
+If the YBA instances are configured to use the HTTPS protocol and you are having problems, verify that certificates and ports are set up correctly. If the issue persists, consider relaxing the certificate validation requirements as a workaround, by enabling the [runtime configuration](../manage-runtime-config/) `yb.ha.ws.ssl.loose.acceptAnyCertificate` (set the flag to `true`).
```

Comment:
Can we add this near line 103 where we talk about the cert setup? It makes more sense as a fallback there.

Reply:
This strikes me more as a troubleshooting step you take after setup and communication is not working (after veryfying HA setup). I think it would worke better in a short Troubleshooting subsection; are there any other tips for troubleshooting your HA setup if it isn't working